src/rules/javascript/eval_string_arg.js

```diff
@@ -0,0 +1,16 @@
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      if (node.callee.property.type === 'Identifier' &&
+          (node.callee.property.name === 'setTimeout' ||
+           node.callee.property.name === 'setInterval')) {
+
+        if (node.arguments.length > 0 &&
+            (node.arguments[0].type !== 'FunctionExpression') &&
```

Comment:
I was thinking about using `indexOf` here against an Array of both types but this is probably more readable.

*I'm making the same comment again because the previous one was buried by changes*